Packs/DigitalGuardian/Integrations/DigitalGuardian/DigitalGuardian.py

```diff
@@ -180,8 +180,8 @@ def check_componentlist_entry():
 
         if 200 <= r.status_code <= 299:
             for jText in json_text:
-                if str(jText['content_value']).lower() == componentlist_entry.lower():
-                    componentlist = jText['content_value']
+                if str(jText.get('content_value')).lower() == componentlist_entry.lower():
```

Comment:
perhaps return a default empty string `jText.get('content_value', '')`
because we are doing lower on that and it will throw exception if returns None